core/include/mmcore/LuaInterpreter.h

```diff
@@ -392,9 +394,14 @@ bool megamol::core::LuaInterpreter<T>::RunString(
                         if (getDouble(1, r)) {
                             result = std::to_string(r);
                         } else {
-                            result = "Result is a complex type.";
-                            megamol::core::utility::log::Log::DefaultLog.WriteError("Lua execution returned complex type");
-                            good = false;
+                            bool b;
+                            if (getBool(1, b)) {
```

Comment:
`to_string(bool)` returns `"0"` or `"1"` and i wonder whether we want `result \in {"true", "false"}` instead.